File: Java/versandetiketten/src/IO/MiddleWareInterface.java
package IO;

import Daten.Adresse;
import Daten.Kunde;

//---------------------------------------------------------------------------------
// Interface zur Middleware
//---------------------------------------------------------------------------------

public interface MiddleWareInterface {
	public Adresse leseAdresse(String adresseID ) throws Exception;
	public Kunde leseKunde(String kundeID ) throws Exception;
}

File: Csharp/VersandEtiketten/IO_Operationen/MiddleWareInterface.cs
using Daten;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IO_Operationen
{
    /// <summary>
    /// Interface zur Middleware
    /// </summary>
    public interface MiddleWareInterface
    {
        public Adresse LeseAdresse(String? adresseID);
        public Kunde LeseKunde(String? kundeID);
    }
}
